### ISSUE TITLE
Scrap failed

### ISSUE THREAD

**Issue by clochix** [y-lohse/cozy-archives#30]
A user is reporting that scraping of http://www.domotique-info.fr/2014/05/recuperer-teleinformation-arduino/ fails.

Here are the logs:

```
Preparing to scrap http://www.domotique-info.fr/2014/05/recuperer-teleinformation-arduino/
domotique-1476257477740tarball transfered to database
./scrap/domotique-1476257477740.tar.gz deleted
Processing of http://www.domotique-info.fr/2014/05/recuperer-teleinformation-arduino/ failed: [object Object]
```

**Comment by y-lohse** [comment]
I managed to reproduce it on my cozy, but as you can see the `failed: [object Object]` error is really not helpful. I'll try uploading a version with better logs to see what is going on.

**Comment by y-lohse** [comment]
I just released version 1.1.3 that includes the fix for this problem :) Thanks for the report!
